src/openrct2/interface/widget.c

```diff
@@ -293,12 +293,14 @@ static void widget_tab_draw(rct_drawpixelinfo *dpi, rct_window *w, sint32 widget
 	if (widget->image == -1)
 		return;
 
+	// Draw outline.
+	widget_draw_image(dpi, w, widgetIndex);
+
 	// Check if tab is disabled
-	if (!widget_is_disabled(w, widgetIndex)) {
-		widget_draw_image(dpi, w, widgetIndex);
+	if (!widget_is_disabled(w, widgetIndex))
 		return;
-	}
 
+	// ??? Seems superfluous, as this function is only used to draw tabs.
```

Comment:
If you look at where this function is called, you will see that its used for:
`WWT_5`, `WWT_COLOURBTN`, `WWT_TRNBTN`, `WWT_TAB`

The function is not well named, as is most of the widget code.